docs/sourcemap-api.asciidoc

```diff
@@ -0,0 +1,84 @@
+[[sourcemap-api]]
+== Sourcemap API
```

Comment:
Should sourcemap really be one word? I've mostly seen it written as two words: source map. If it should be source map, please apply this edit globally.